### ISSUE TITLE
UndoRedo behavior method not called

### ISSUE THREAD

**Issue by AlexHolly** [godotengine/godot#18744]
Just started using Undo Redo two things I noticed (maybe I use it wrong since I couldn't find a proper tutorial)

1. Methods with "null" are not called
Works in c++? https://github.com/godotengine/godot/blob/6d9cc032e71ccc768064bf908623b0954d01181c/editor/scene_tree_dock.cpp#L1667
```
func _ready():
	var undo_redo = UndoRedo.new()
	undo_redo.create_action("Test")
	undo_redo.add_do_method(self, "test", 42)
	undo_redo.add_undo_method(self, "test", null) # This is never called
	undo_redo.commit_action()
	
	undo_redo.redo()
	undo_redo.undo()

func test(v):
	print("++")
	print(v)
```

2. Non existent methods don't throw an error
```
func _ready():
	var undo_redo = UndoRedo.new()
	undo_redo.create_action("Test")
	undo_redo.add_do_method(self, "test", 42)
	undo_redo.add_undo_method(self, "test1", 42) # Throws no error
	undo_redo.commit_action()
	
	undo_redo.redo()
	undo_redo.undo()

func test(v):
	print(v)
```

**Comment by reduz** [comment]
This should probably happen for the editor too, given it could help fix issues,
I would replace this call:
https://github.com/godotengine/godot/blob/master/core/undo_redo.cpp#L271
for the other call method in Object, which can return a Variant::CallError, which can be printed properly with:
```
static String Variant::get_call_error_text(Object *p_base, const StringName &p_method, const Variant **p_argptrs, int p_argcount, const Variant::CallError &ce);
```

**Comment by akien-mga** [comment]
Fixed by #21856.
